frontend/src/templates/Field/Table/TableField.tsx

```diff
@@ -109,7 +109,20 @@ export const TableField = ({
 
   return (
     <TableFieldContainer schema={schema}>
-      <Box d="block" w="100%" overflowX="auto">
+      <Box
+        d="block"
+        w="100%"
+        overflowX="auto"
+        sx={{
+          '&::-webkit-scrollbar': {
+            height: '7px',
+          },
+          '&::-webkit-scrollbar-thumb': {
+            background: 'rgba(0,0,0,.5)',
+            borderRadius: '4px',
+          },
+        }}
```

Comment:
nit: Should we extract these styles to an util/constant so it can be reused (if needed)?

Reply:
perhaps if its gonna be used in multiple places, feel like this is a pretty niche scenario